Backend/admin_dashboard/serializers.py

```diff
@@ -75,4 +75,26 @@ def to_representation(self, instance):
         data = super().to_representation(instance)
         if data.get('user') is None:
             data.pop('username', None)
-        return data
\ No newline at end of file
+        return data
+
+class Types_serializer(serializers.ModelSerializer):
+    class Meta:
+        model = Type 
+        fields = (
+            "id",
+            "name",
```

Comment:
nie do konca name a raczej value bo name to ja bede podpinal po froncie ale obstawiam ze o to ci tutaj chodzilo